### PR TITLE
[MIG] sale_coupon_criteria_multi_product: Migration to 14.0

### PR DESCRIPTION
**Pull request by alvaro-domatix**

### CODE REVIEW

File: sale_coupon_criteria_multi_product/models/sale_coupon_program.py [hunk 1]
```diff
@@ -4,26 +4,26 @@
 
 
 class SaleCouponProgram(models.Model):
-    _inherit = "sale.coupon.program"
+    _inherit = "coupon.program"
 
-    sale_coupon_criteria = fields.Selection(
+    coupon_criteria = fields.Selection(
```
Comment (chienandalu) [inline sale_coupon_criteria_multi_product/models/sale_coupon_program.py]: Ok with the field renames (you can also do it with the filenames using git mv for better diff). Anyway, a migration script should be made. An example:

An example: https://github.com/OCA/account-financial-tools/blob/13.0/account_move_line_sale_info/migrations/13.0.1.0.0/pre-migration.py

`openupgrade.rename_fields` reference: https://github.com/OCA/openupgradelib/blob/master/openupgradelib/openupgrade.py#L585-L611

**Comment by alvaro-domatix** [comment]
Done!

**Review by francesco-ooops — APPROVED** [review]
functionally LGTM

**Review by chienandalu — APPROVED** [review]
Thanks @alvaro-domatix !

**Pull request merged**